src/levels.rs

```diff
@@ -308,7 +309,7 @@ impl Default for Av1Block_intra_inter {
     }
 }
 
-#[derive(Default)]
+#[derive(Default, Clone, Copy)]
```

Comment:
This is a bit big to be `Copy`.  Why do we need it to be `Copy`?  The `union` fields are only `Copy` because they have to be.  If it's because of `vec![]`, we can write it as `.map` and `.collect` instead.

Reply:
Fair point. I needed the type to be `Clone` so that I could use `vec!` to initialize the field, but I don't need it to be copy. Will remove the derive.

Reply:
You don't need to use `vec![]`, though, that was my point.

Reply:
Yup, I realized that after writing the comment, sorry. I forgot to remove the derive on `Av1Block_intra_inter` but that's fixed now.